rainforest/tests_test.go

```diff
@@ -403,6 +416,8 @@ func TestUpdateTest(t *testing.T) {
 			t.Errorf("Unexpected browsers received. Expected: [], Got: %v", bodyStr)
 		} else if !strings.Contains(bodyStr, "\"tags\":[]") {
 			t.Errorf("Unexpected tags received. Expected: [], Got: %v", bodyStr)
+		} else if !strings.Contains(bodyStr, "\"folder_id\":null") {
```

Comment:
Is there already a test that it *doesn't* include this if `folder_id` is 0?

Reply:
Yep, line 338 has that test.